### PR TITLE
[temp] - add support for prefix ESDT

### PR DESCRIPTION
**Pull request by cfaur09**
## Reasoning
- 
- 
- 
  
## Proposed Changes
- 
- 
- 

## How to test
- 
- 
-

### CODE REVIEW

**Review by bogdan-rosianu — APPROVED** [review]

File: src/endpoints/collections/collection.service.ts [hunk 1]
```diff
@@ -51,6 +51,7 @@ export class CollectionService {
 
   async getNftCollections(pagination: QueryPagination, filter: CollectionFilter): Promise<NftCollection[]> {
     const tokenCollections = await this.indexerService.getNftCollections(pagination, filter);
+    console.log(tokenCollections);
```
Comment (dragos-rebegea) [inline src/endpoints/collections/collection.service.ts:54]: expected?

File: src/endpoints/tokens/token.service.ts [hunk 2]
```diff
@@ -541,11 +550,13 @@ export class TokenService {
     let circulatingSupply: string | number;
 
     const properties = await this.getTokenProperties(identifier);
+    console.log(properties);
```
Comment (dragos-rebegea) [inline src/endpoints/tokens/token.service.ts:553]: expected?

File: src/endpoints/tokens/token.service.ts [hunk 3]
```diff
@@ -541,11 +550,13 @@ export class TokenService {
...
     if (!properties) {
       return undefined;
     }
 
     const result = await this.esdtService.getTokenSupply(identifier);
+    console.log(result);
```
Comment (dragos-rebegea) [inline src/endpoints/tokens/token.service.ts:559]: expected?

File: src/endpoints/tokens/token.service.ts (hunk 2 above)
Comment (bogdan-rosianu) [reply src/endpoints/tokens/token.service.ts]: removed

File: src/endpoints/tokens/token.service.ts (hunk 3 above)
Comment (bogdan-rosianu) [reply src/endpoints/tokens/token.service.ts]: removed

**Review by bogdan-rosianu — DISMISSED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Review by dragos-rebegea — APPROVED** [review]

**Pull request merged**